File: Javascript/1002.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let pi = 3.14159;
let radius = Number(lines.shift());
let area = pi * Math.pow(radius,2); 
console.log("A=" + area.toFixed(4));

File: C/1002.c
#include <stdio.h>
#include <math.h> 
 
int main() {
 
    double radius, pi = 3.14159;

    scanf("%lf", &radius);
    printf("A=%.4f\n",  pi * pow(radius,2));
 
    return 0;
}